client/me/purchases/manage-purchase/plan-details/billing-period.jsx

```diff
@@ -21,11 +21,13 @@ import { getYearlyPlanByMonthly } from 'lib/plans';
 import { planItem } from 'lib/cart-values/cart-items';
 import { addItem } from 'lib/cart/actions';
 import { isExpired, isExpiring, isRenewing, showCreditCardExpiringWarning } from 'lib/purchases';
+import { JETPACK_SUPPORT } from 'lib/url/support';
 import { recordTracksEvent } from 'state/analytics/actions';
 
-class PlanBillingPeriod extends Component {
+export class PlanBillingPeriod extends Component {
```

Comment:
Was this necessary for the tests, I guess?  (i.e. the default export doesn't work there for some reason?)

I'm asking mainly because it looks like one test is using the default import and the other is taking advantage of this change.

Reply:
Yes, I had to add this because the default export is a connected component, which requires a parent Redux context to work. This way we can test the component and don't worry about Redux.